shapeworks_cloud/core/deepssm_tasks.py

```diff
@@ -0,0 +1,557 @@
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Dict
+
+import DataAugmentationUtils
+import DeepSSMUtils
+from celery import shared_task
+from django.conf import settings
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+import shapeworks as sw
+
+from shapeworks_cloud.core import models
+from swcc.api import swcc_session
+from swcc.models import Project as SWCCProject
+
+from .tasks import edit_swproj_section
+
+
+def interpret_deepssm_form_data(data: Dict) -> Dict:
+    # Prep
+    data['validation_split'] = data['validationSplit']
```

Comment:
The camelCase keys are coming from the frontend request. I think these can be consolidated to match. This is mainly in place because I didn't have which deepssm fields were being used by the app, so I had to add them myself and consolidate there.

Reply:
Fixed: [0c0b918](https://github.com/girder/shapeworks-cloud/pull/337/commits/0c0b9182bb5416c8e26357f8d48f9ee5e18e30d0)